src/constants.rs

```diff
@@ -43,6 +43,33 @@ pub const SCHNORRSIG_PUBLIC_KEY_SIZE: usize = 32;
 /// Size of a key pair
 pub const KEY_PAIR_SIZE: usize = 96;
 
+/// The size of a generator point
+pub const GENERATOR_SIZE: usize = 64;
+
+/// The size of a Pedersen commitment
+pub const PEDERSEN_COMMITMENT_SIZE: usize = 33;
+
+/// The size of a Pedersen commitment
+pub const PEDERSEN_COMMITMENT_SIZE_INTERNAL: usize = 64;
+
+/// The size of a single Bullet proof
+pub const SINGLE_BULLET_PROOF_SIZE: usize = 675;
+
+#[cfg(feature = "bullet-proof-sizing")]
+pub const MAX_PROOF_SIZE: usize = SINGLE_BULLET_PROOF_SIZE;
+/// The max size of a range proof
+#[cfg(not(feature = "bullet-proof-sizing"))]
+pub const MAX_PROOF_SIZE: usize = 5134;
+
+/// The maximum size of a message embedded in a range proof
+#[cfg(not(feature = "bullet-proof-sizing"))]
+pub const PROOF_MSG_SIZE: usize = 2048;
+#[cfg(feature = "bullet-proof-sizing")]
+pub const PROOF_MSG_SIZE: usize = 2048;
```

Comment:
Why the conditional compilation with the same values?

Reply:
This is directly taken from the base repo https://github.com/mimblewimble/rust-secp256k1-zkp. I agree that it does not make sense for me too. I'd think about removing the `not` case.

Reply:
I'm guessing this is a holdover from an old change somewhere. Presumably they used to have different values.
Should be no problem removing this conditional logic.